src/components/molecules/pager/pager.jsx

```diff
@@ -0,0 +1,214 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import './pager.scss';
```

Comment:
ack, sorry I didn't see this sooner.  Can we move the styling to app.scss?  Thinking about this as a component in a component library and styling wouldn't be married to the component

Reply:
Done!